packages/create-app/src/prompts/init.test.ts

```diff
@@ -23,7 +24,9 @@ describe('init', () => {
         name: 'name',
         preface: '\nWelcome. Let’s get started by naming your app. You can change it later.',
         message: "Your app's name?",
-        default: expect.stringMatching(/^\w+-\w+-app-\d+$/),
+        default: expect.stringMatching(
+          new RegExp(`^${haiku.SAFE_ADJECTIVES.join('|')}-${haiku.SAFE_NOUNS.join('|')}-app$`),
+        ),
```

Comment:
since you are adding specific tests to `haiku.ts`, there is no need to test again here that the generated values are "valid". 
I'd revert this test, and maybe make `SAFE_ADJECTIVES` and `SAFE_NOUNS` private if possible